src/web/start_bot.py

```
@@ -3,19 +3,13 @@
-try:
-    from svg_translate import get_files, get_wikitext, start_injects, extract
-except ImportError:
-    from src.svg_translate import (
-        get_files,
-        get_wikitext,
-        start_injects,
-        extract,
-    )
+from svg_translate import get_files, get_wikitext, start_injects, extract
+
 from .download_task import download_one_file
```

Pylint (4.0.1)

[error] Attempted relative import beyond top-level package

(E0402)